README.md

```diff
@@ -31,40 +33,43 @@ High level diagram of the stack:
+| Package                                                | Version                         | Description                                                                          |
+| ------------------------------------------------------ | ------------------------------- | ------------------------------------------------------------------------------------ |
+| [opensearch-single](katalog/opensearch-single)         | `2.12.0`                        | Single node opensearch deployment. Not intended for production use.                  |
+| [opensearch-triple](katalog/opensearch-triple)         | `2.12.0`                        | Three node high-availability opensearch deployment                                   |
+| [opensearch-dashboards](katalog/opensearch-dashboards) | `2.12.0`                        | Analytics and visualization platform for Opensearch                                  |
+| [logging-operator](katalog/logging-operator)           | `4.5.6`                         | Banzai logging operator, manages fluentbit/fluentd and their configurations          |
+| [logging-operated](katalog/logging-operated)           | `-`                             | fluentd and fluentbit deployment using logging operator                              |
+| [configs](katalog/configs)                             | `-`                             | Logging pipeline configs to gather various types of logs and send them to OpenSearch |
+| [loki-configs](katalog/loki-configs)                   | `-`                             | Logging pipeline configs to gather various types of logs and send them to Loki       |
+| [loki-distributed](katalog/loki-distributed)           | `2.9.2`                         | Distributed Loki deployment                                                          |
+| [minio-ha](katalog/minio-ha)                           | `vRELEASE.2023-01-12T02-06-16Z` | Three nodes HA MinIO deployment                                                      |
 
 Click on each package to see its full documentation.
 
 ## Compatibility
 
 | Kubernetes Version |   Compatibility    | Notes           |
 | ------------------ | :----------------: | --------------- |
-| `1.25.x`           | :white_check_mark: | No known issues |
 | `1.26.x`           | :white_check_mark: | No known issues |
 | `1.27.x`           | :white_check_mark: | No known issues |
+| `1.28.x`           | :white_check_mark: | No known issues |
+| `1.29.x`           | :white_check_mark: | No known issues |
 
-Check the [compatibility matrix][compatibility-matrix] for additional informations about previous releases of the modules.
+Check the [compatibility matrix][compatibility-matrix] for additional information about previous releases of the modules.
 
 ## Usage
 
 ### Prerequisites
 
+| Tool                        | Version    | Description                                                                                                                                                    |
```

Comment:
I think this is a double header

Reply:
I don't know how that ended up in there, fixed!